Executables/Game/World/Entities/PlayerEntity.cs

```diff
@@ -173,7 +173,7 @@ private static uint GetMaxHp(IJobManager jobManager, byte playerClass, byte leve
 
         public async Task Load()
         {
-            Empire = await _empireRepository.GetEmpireForAccountAsync(Player.AccountId) ?? 0;
+            Empire = await _empireRepository.GetEmpireForPlayerAsync(Player.AccountId) ?? 0;
```

Comment:
You should put `Player.Id` instead of `Player.AccountId` because you retrieve the empire by the player ID not by the account ID